Assets/MRTK/SDK/Experimental/Elastic/Scripts/ElasticHandleLogic.cs

```diff
@@ -0,0 +1,122 @@
+        {
+            // If we have not been Setup() yet
+            if (!isSetup) { return 0.1f; }
+
+            var handleAxis = normalizedAxis ?? (rightInitialPosition - leftInitialPosition).normalized;
+
+            var handDistance = elasticSystem.GetCurrentValue();
+            if (leftPointer.HasValue && rightPointer.HasValue)
+            {
+                // If we have both pointers, we simply project the vector between the
+                // two hands onto the handle axis vector.
+                handDistance = Vector3.Dot(rightPointer.Value - leftPointer.Value, handleAxis);
+            }
+            else if (!leftPointer.HasValue && rightPointer.HasValue)
+            {
+                // If we only have a right pointer, calculate the hand distance
+                // as twice the distance of the right pointer from the center.
+                handDistance = 2.0f * Vector3.Dot(rightPointer.Value - (leftInitialPosition + rightInitialPosition) / 2.0f, handleAxis);
+            }
+            else if (leftPointer.HasValue && !rightPointer.HasValue)
+            {
+                // If we only have a left pointer, calculate the hand distance
+                // as twice the distance of the left pointer from the center.
+                handDistance = 2.0f * -Vector3.Dot(leftPointer.Value - (leftInitialPosition + rightInitialPosition) / 2.0f, handleAxis);
+            }
+
+            return elasticSystem.ComputeIteration(handDistance, deltaTime);
+        }
+
+        /// <summary>
+        /// Computes the "center position" between the two "hands". However, this
+        /// includes some extra checks; if the hand is not present (i.e., the argument
+        /// has been nullified) this method will substitude the leftInitialPosition or
+        /// rightInitialPosition in place of the missing hand. This is for accessibility
+        /// purposes, so single-handed elastic handle logic can be implemented.
+        /// </summary>
+        /// <param name="leftPointer">Nullable position of left hand/grab point</param>
+        /// <param name="rightPointer">Nullable position of right hand/grab point</param>
+        /// <returns>Calculated center point, according to the above criteria</returns>
+        public virtual Vector3 GetCenterPosition(Vector3? leftPointer, Vector3? rightPointer)
+        {
+            if (leftPointer.HasValue && rightPointer.HasValue)
+            {
+                return (leftPointer.Value + rightPointer.Value) / 2.0f;
+            }
+            else if (!leftPointer.HasValue && rightPointer.HasValue)
+            {
+                return (rightPointer.Value + leftInitialPosition) / 2.0f;
+            }
+            else if (leftPointer.HasValue && !rightPointer.HasValue)
+            {
+                return (leftPointer.Value + rightInitialPosition) / 2.0f;
+            } else
```

Comment:
Move else to its own line
}
else
{
}

Reply:
Fixed, thank you